src/utils.h

```diff
@@ -245,5 +245,23 @@ FILE * fopen_verbose(const char * const path);
  * when the environment variable doesn't exits.
  */
 void add_paths_from_env(GPtrArray *arr, char *env_name, char *subdir, char *alternative);
+
+/**
+ * Hexdumps buffer, 1 line = 8 bytes with character representation
+ *
+ * @param buffer Buffer to hexdump
+ * @param len Length of the buffer
+ */
+void hexdump(char * buffer, int len);
```

Comment:
Why don't we convert to int string instead? `printf("%i", val)`

Reply:
?

Length is for binary buffer boundaries (so it can hexdump NULLs) - why would we need string representation of the buffer length?